WinUIGallery/ItemPage.xaml.cs

```diff
@@ -33,6 +33,9 @@ namespace AppUIBasics
     /// </summary>
     public sealed partial class ItemPage : Page
     {
+        private static string WinUIBaseUrl = "https://github.com/microsoft/microsoft-ui-xaml/tree/winui3/release/1.4-stable/controls/dev/";
```

Comment:
@karkarl @chingucoding Same here. Wish we could use a `aka.ms/latestwinuipath` instead and then append the `SourcePath`, but that does not seem to work :(

Reply:
The more I think about it, do we really want to link to the absolutly latest hot of the press release or the one people are seeing in the app? I would argue, if I see a control in the WinUI Gallery, I want to see the source code that made the control the way it is in the gallery and not the one inside the latest release. Otherwise, it might be a confusing experience when copying it over into one's own code and the control looking different then.